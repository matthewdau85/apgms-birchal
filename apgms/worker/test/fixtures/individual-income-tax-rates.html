<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <title>Individual income tax rates</title>
  </head>
  <body>
    <h1>Individual income tax rates</h1>
    <p>These rates apply from 1 July 2024.</p>
    <table>
      <thead>
        <tr>
          <th>Taxable income</th>
          <th>Tax on this income</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>$0 – $18,200</td>
          <td>Nil</td>
        </tr>
        <tr>
          <td>$18,201 – $45,000</td>
          <td>19c for each $1 over $18,200</td>
        </tr>
        <tr>
          <td>$45,001 – $120,000</td>
          <td>$5,092 plus 32.5c for each $1 over $45,000</td>
        </tr>
        <tr>
          <td>$120,001 – $180,000</td>
          <td>$29,467 plus 37c for each $1 over $120,000</td>
        </tr>
        <tr>
          <td>$180,001 and over</td>
          <td>$51,667 plus 45c for each $1 over $180,000</td>
        </tr>
      </tbody>
    </table>
  </body>
</html>
