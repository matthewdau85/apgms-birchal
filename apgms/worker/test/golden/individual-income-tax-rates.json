{
  "slug": "individual-income-tax-rates",
  "effectiveFrom": "2024-07-01",
  "brackets": [
    {
      "index": 0,
      "lower": 0,
      "upper": 18200,
      "marginalRate": 0,
      "baseTax": 0,
      "threshold": 0
    },
    {
      "index": 1,
      "lower": 18201,
      "upper": 45000,
      "marginalRate": 0.19,
      "baseTax": 0,
      "threshold": 18200
    },
    {
      "index": 2,
      "lower": 45001,
      "upper": 120000,
      "marginalRate": 0.325,
      "baseTax": 5092,
      "threshold": 45000
    },
    {
      "index": 3,
      "lower": 120001,
      "upper": 180000,
      "marginalRate": 0.37,
      "baseTax": 29467,
      "threshold": 120000
    },
    {
      "index": 4,
      "lower": 180001,
      "upper": null,
      "marginalRate": 0.45,
      "baseTax": 51667,
      "threshold": 180000
    }
  ],
  "sourceUrl": "https://www.ato.gov.au/rates/individual-income-tax-rates/"
}
